### PR TITLE
Tuned profile hierarchy change.

### PR DESCRIPTION
**Pull request by jmencak**
Documentation based on https://github.com/openshift/openshift-ansible/pull/5054 and https://github.com/openshift/openshift-ansible/pull/5089. See the former for a diagram.  Also changed TuneD --> **Tuned** based on RHEL documentation.

https://trello.com/c/Bw3Sk0Jt/624-document-scale-37-ocp-spike-update-tuned-profile-hierarchy-for-37

### CODE REVIEW

File: scaling_performance/host_practices.adoc [hunk 1]
```diff
@@ -137,32 +137,39 @@ etcd processes are typically memory intensive. Master / API server processes are
 ====
 
 [[scaling-performance-capacity-tuned-profile]]
-== Scaling Hosts Using the TuneD Profile
+== Scaling Hosts Using the Tuned Profile
 
-TuneD is a tuning profile delivery mechanism enabled by default in Red Hat
-Enterprise Linux and other Red Hat products. TuneD customizes Linux settings,
+**Tuned** is a tuning profile delivery mechanism enabled by default in Red Hat
```
Comment (ahardin-rh) [inline scaling_performance/host_practices.adoc]: @jmencak In the RHEL docs, Tuned is consistently referred to as the **tuned-adm** tool. Should that be the case across this topic, or is calling it **Tuned** most appropriate?

**Comment by ahardin-rh** [comment]
@jmencak Thanks so much! Just one question for you.

File: scaling_performance/host_practices.adoc (hunk 1 above)
Comment (jmencak) [reply scaling_performance/host_practices.adoc]: tuned-adm tool is just the Tuned client.  Imagine it as a subset of Tuned.  Tuned is the whole package/system of setting profiles.

Comment (ahardin-rh) [reply scaling_performance/host_practices.adoc]: @jmencak Got it. Thanks for clarifying!

**Comment by ahardin-rh** [comment]
Merging this and will apply very minor style edits in a separate PR.

**Pull request merged**